src/js/easymde.js

```diff
@@ -2025,7 +2042,7 @@ EasyMDE.prototype.autosave = function () {
         if (el != null && el != undefined && el != '') {
             var d = new Date();
             var dd = new Intl.DateTimeFormat([this.options.autosave.timeFormat.locale, 'en-US'], this.options.autosave.timeFormat.format).format(d);
-            var save = this.options.autosave.text == undefined ? 'Autosaved: ' : this.options.autosave.text;
```

Comment:
You change this one

Reply:
But how does this affect work?

Reply:
If someone use `this.options.autosave.text` in this setting, this will not work anymore because you change to `this.options.statusTexts.autosave`. Old settings, should still work after your change.

Reply:
It will work, just reset to the default value.
var statusTexts = {
    lines: 'lines: ',
    words: 'words: ',
    autosave: 'Autosaved: ',
};

Reply:
If the user adds translation of statuses, what prevents them from updating the AutoSave text? Despite the fact that it appeared recently.

Reply:
> It will work, just reset to the default value.
> var statusTexts = {
> lines: 'lines: ',
> words: 'words: ',
> autosave: 'Autosaved: ',
> };

But we should keep the user setting not the default setting, because this make a breakchange.